app/models/user.rb

```diff
@@ -50,6 +50,13 @@ def groups
     []
   end
 
+  def as_json(*)
```

Comment:
I added this model method because in `Hyrax::UsersControllerBehavior` it calls `.to_json` on a collection of `User` objects. ([source](https://github.com/projecthydra-labs/hyrax/blob/master/app/controllers/concerns/hyrax/users_controller_behavior.rb#L17)). I wanted a way to change the JSON output without overriding anything in Hyrax from Hyku.

Should Hyrax be changed to use JBuilder instead?